data/json/items/armor/legs_clothes.json

```diff
@@ -606,7 +606,377 @@
+      },
+      {
+        "id": "pants_desert",
+        "name": { "str": "desert-camo pants", "str_pl": "pairs of desert-camo pants" },
+        "description": "Dyed a mix of brown, tan, and yellowish beige hues, this pair of hardy trousers is designed to seamlessly blend into arid environments.  That being said, the fact that New England is famously devoid of deserts means that the pattern’s only utility is in looking cool.",
+        "weight": 6
+      },
+      {
+        "id": "pants_alpine",
+        "name": { "str": "alpine-camo pants", "str_pl": "pairs of alpine-camo pants" },
+        "description": "The companion of aspiring mountaineers and alpine hunters, these brown, gray, and white dyed pants possess a network of jagged shapes and mottled lines, designed to break up the wearer’s profile and better conceal them against the background landscape of mountains.",
+        "weight": 6
+      },
+      {
+        "id": "pants_urban",
+        "name": { "str": "urban-camo pants", "str_pl": "pairs of urban-camo pants" },
+        "description": "A set of rugged yet comfortable pants dyed in various scales of gray, black, and dirty white, possessing a set of sizable pockets, and printed with squares and irregular polygons to disrupt a wearer’s silhouette within man-made environments.  You would honestly not be surprised if you learned that this singular pair of trousers contains 50 shades of gray.",
+        "weight": 6
+      },
+      {
+        "id": "pants_ocean",
+        "name": { "str": "ocean-camo pants", "str_pl": "pairs of ocean-camo pants" },
+        "description": "Printed with a camouflage pattern most commonly associated with naval vessels or aircraft, this pair of pants is festooned with a wide array of blue hues and disruptive lines, with a touch of light gray also making its presence known.  Warmer than jeans and possessing a number of deep pockets, this comfortable pair of trousers would do an admirable job of concealing you in maritime environments.  You have yet to come up with a reason why you’d ever be in a maritime environment.",
+        "weight": 6
+      },
+      {
+        "id": "pants_river",
+        "name": { "str": "river-camo pants", "str_pl": "pairs of river-camo pants" },
+        "description": "Employed by anglers and river-side hunters for camouflage in freshwater habitats, this pair of bootcut trousers incorporates shades of green, brown, blue, and gray, along with sizable pockets and a construction favoring heat retention.",
+        "weight": 6
+      },
+      {
+        "id": "pants_arctic",
+        "name": { "str": "arctic-camo pants", "str_pl": "pairs of arctic-camo pants" },
+        "description": "Featuring a camouflage pattern consisting of white, shades of gray, and a slight tinge of brown, this pair of rugged pants is fit for snowy tundra and poler regions—or it would be, had the garment not been only moderately warmer than jeans.  You have yet to grasp their point for existing.",
```

Comment:
```suggestion
        "description": "Featuring a camouflage pattern consisting of white, shades of gray, and a slight tinge of brown, this pair of rugged pants is fit for snowy tundra and polar regions—or it would be, had the garment not been only moderately warmer than jeans.  You have yet to grasp their point for existing.",
```